selenium-testing/README.md

```diff
@@ -0,0 +1,21 @@
+# Selenium Testing
+
+Instructions on how to install and run selenium tests
+
+## Environment Setup - Eclipse IDE
+
+**`Step 1`** Download JDK 21
+
+**`Step 2`** Download the Eclipse IDE
+
+**`Step 3`** On the Eclipse marketplace, install the **TestNG for Eclipse** plugin
+
+## Run tests
+
+First, rename the file `config.properties.template` to `config.properties`, then fill in the missing details with base64 string encoded values
```

Comment:
You probably want to change this step to copy and name as. If you rename it'll show up as a delete in git.

Reply:
Ohh I see: https://github.com/bcgov/ecc-ofm/pull/319/commits/9414a142742a844d2b6f841143a9e6c034e190a6